includes/discovery/sensors/entity-sensor.inc.php

```diff
@@ -50,7 +50,7 @@
                 $descr = rewrite_entity_descr($descr);
             }
 
-            $thisisnotbullshit = true;
+            $thisisnotbullshit = check_entity_sensor($descr, $device);
```

Comment:
Can you substitute this variable name? Its really not appealing and makes me remember the tyranny of the unspoken one